File: src/Frontend/heartmade-candles/src/types/CandleDetail.ts
import { Candle } from './Candle'
import { Decor } from './Decor'
import { LayerColor } from './LayerColor'
import { NumberOfLayer } from './NumberOfLayer'
import { Smell } from './Smell'
import { Wick } from './Wick'

export interface CandleDetail {
  candle: Candle;
  decors: Decor[];
  layerColors: LayerColor[];
  numberOfLayers: NumberOfLayer[];
  smells: Smell[];
  wicks: Wick[];
}

File: src/HeartmadeCandles/HeartmadeCandles.Constructor.Core/Models/CandleDetail.cs
namespace HeartmadeCandles.Constructor.Core.Models
{
    public class CandleDetail
    {
        public Candle Candle { get; set; }
        public Decor[] Decors { get; set; }
        public LayerColor[] LayerColors { get; set; }
        public NumberOfLayer[] NumberOfLayers { get; set; }
        public Smell[] Smells { get; set; }
        public Wick[] Wicks { get; set; }
    }
}
